components/common/HeadTags.tsx

```diff
@@ -0,0 +1,60 @@
+import Head from 'next/head'
+
+const defaultTitle = 'The Stories of COVID-19 in Ontario'
+const defaultDescription =
+  "Ontario is in a humanitarian crisis. If our leaders won't listen to the numbers, they must face our stories."
+const defaultPreviewImage = `img/landingpage-v2.jpg`
+
+interface HeadTagsProps {
+  title?: string
+  description?: string
+  previewImage?: string
+}
+
+const TITLE_SUFFIX = 'MyCOVIDStory.ca'
+const DESCRIPTION_LENGTH = 170
+
+export default function HeadTags({
```

Comment:
Thoughts on letting this accept children to pass through? For example a noindex meta tag?

Reply:
Ah, yeah - good call.